lib/epi_locator_web/templates/tr/person_search.xml.eex

```diff
@@ -39,7 +39,7 @@
     </Criteria>
     <Datasources>
         <PublicRecordPeople>true</PublicRecordPeople>
-        <NPIRecord>true</NPIRecord>
-        <WorldCheckRiskIntelligence>true</WorldCheckRiskIntelligence>
+        <NPIRecord>false</NPIRecord>
```

Comment:
Undocumented change from v2 to v3. v2 would silently ignore these if they were disabled based on the products paid for by customer. v3 will return an error, so I'm specifically disabling.